vsr: simplify empty checksum

matklad opened the pull request:
No _real_ motivation here, it just feels awkward to do runtime synchronization here.

The test is right next to `checksum_body_empty`, so, if any of our tests use this constant, the test is guaranteed to be included.

matklad committed:
vsr: simplify empty checksum

No _real_ motivation here, it just feels awkward to do runtime
synchronization here.

The test is right next to `checksum_body_empty`, so, if any of our tests
use this constant, the test is guaranteed to be included.

kprotty commented:
Neat, is there a way to do something like this for `Environment.members()` in `superblock_fuzz.zig` or is that not needed?

matklad committed:
vsr: simplify superblock fuzz

No need to once this when we can inject `members`. If we end up
generating unique replica ids with crypto.random, it would even make
sense to generate unique members.

matklad commented:
I guess so, ptal!
